internal/users/localentries/lockedentries.go

```diff
@@ -62,6 +62,13 @@ var defaultOptions = options{
 // Option represents an optional function to override [UserDBLocked] default values.
 type Option func(*options)
 
+type invalidEntry struct {
```

Comment:
It was the very same idea I had initially... Then I thought it was too much to add an extra type just for it (given you asked me to keep it simple 😅), but this is what I also had in mind so I'm fine.